bench/workgen/workgen.cpp

```diff
@@ -1328,37 +1443,84 @@ ThreadRunner::op_get_key_recno(Operation *op, uint64_t range, tint_t tint)
     return (rval % recno_count + 1); // recnos are one-based.
 }
 
+// Clear the table uri and tint value for a dynamic table operation.
+void
+ThreadRunner::op_clear_table(Operation *op)
+{
+    if (op->_random_table && op->is_table_op()) {
+        op->_table._uri = std::string();
+        op->_table._internal->_tint = 0;
+    }
+}
+
+// Generate and set a key and value for the operation managed by this thread.
+// Set the table uri and tint value for a dynamic table operation.
```

Comment:
Looks like this comment applies to the `op_kv_gen` function, not this one?

Reply:
Yes it does, I missed it when shuffling code around. Fixed in [74294b2](https://github.com/wiredtiger/wiredtiger/pull/8798/commits/74294b2b7c633043571571c79b4d43a7f71ff7ba).